src/fileoverview_comment_transformer.ts

```diff
@@ -0,0 +1,79 @@
+import * as ts from 'typescript';
+
+import * as jsdoc from './jsdoc';
+import {createNotEmittedStatement, updateSourceFileNode} from './transformer_util';
+
+/**
+ * A set of JSDoc tags that mark a comment as a fileoverview comment. These are recognized by other
+ * pieces of infrastructure (Closure Compiler, module system, ...).
+ */
+const FILEOVERVIEW_COMMENT_MARKERS: ReadonlySet<string> =
+    new Set(['fileoverview', 'externs', 'modName', 'mods', 'pintomodule']);
+
+/**
+ * A transformer that ensures the emitted JS file has an \@fileoverview comment that contains a
+ * \@suppress {checkTypes} annotation by either adding or updating an existing comment.
```

Comment:
an @fileoverview vs
a @suppress
pick one of a/an

Reply:
Done.